cloudwatch/cloudwatch.go

```diff
@@ -608,16 +618,16 @@ func (output *OutputPlugin) processRejectedEventsInfo(response *cloudwatchlogs.P
 // not constitute valid UTF-8 encoded Unicode codepoints with the Unicode
 // replacement codepoint U+FFFD (a 3-byte UTF-8 sequence, represented in Go as
 // utf8.RuneError)
-func effectiveLen(line string) int {
-	effectiveBytes := 0
-	for _, rune := range line {
-		effectiveBytes += utf8.RuneLen(rune)
-	}
-	return effectiveBytes
-}
+// func effectiveLen(line string) int {
```

Comment:
Should we delete it as we are not using it?

Reply:
Shall I delete it at all? I just comment it right now

Reply:
I prefer removing dead code.

Reply:
␡

Reply:
sure